client/src/pages/ManageBookings/CalenderView.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react';
+import useStyles from './useStyles'
+import { takeMonth }from './Calender'
+import { Button, Grid, IconButton, Paper } from '@mui/material'
+import {format, isSameMonth, isSameDay, addMonths, subMonths} from 'date-fns'
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+
+
+// const minDate = new Date('2020-01-01T00:00:00.000');
+// const maxDate = new Date('2034-01-01T00:00:00.000');
+
+
+
+const CalenderView = () => {
+    const classes = useStyles();
+    const [selectedDate, setSelectedDate] = useState(new Date());
+    const data = takeMonth(selectedDate)();
+
+    function dayColor(day:Date){
+      if(!isSameMonth(day, selectedDate)) return '#CCCCCC';
+      if(isSameDay(day, selectedDate)) return '#FFFFFF';
+      return '#000000';
+    }
+
+    function dayBGColor(day:Date){
+      if(!isSameMonth(day, selectedDate)) return '#FFFFFF';
+      if(isSameDay(day, selectedDate)) return '#f14140';
```

Comment:
Okay, so you have the right idea but the red background is actually showing days they have bookings available